Rakefile

```diff
@@ -5,3 +5,49 @@ begin
   task :default => :spec
 rescue LoadError
 end
+
+ENV["CODECLIMATE_DEBUG"] = "1" if ENV["DEBUG"]
+
+IMAGE_NAME = "codeclimate/codeclimate-duplication"
+
+task :image do
+  sh "docker build --rm -t #{IMAGE_NAME} ."
```

Comment:
We prefer to use `make` for these - this repo already has a `Makefile`. Can you move tasks there if they don't already exist?